desktop/ui/src/main/java/org/datacleaner/windows/ResultWindow.java

```diff
@@ -359,15 +398,18 @@ protected JComponent getWindowContent() {
 
         final DCBannerPanel banner = new DCBannerPanel(imageManager.getImage("images/window/banner-results.png"),
                 bannerTitle);
-        _tabbedPane.bindTabTitleToBanner(banner);
+        _tabbedPane.addListener(new VerticalTabbedPane.Listener() {
+            @Override
+            public void stateChanged(int newIndex, Tab<?> newTab) {
+                banner.setTitle2(newTab.getTitle());
+                banner.updateUI();
+            }
+        });
 
         for (JComponent pluggableButton : _pluggableButtons) {
             banner.add(pluggableButton);
```

Comment:
I would suggest the following:

 * Add a new field for pluggable menu items
 * Deprecate the old field for pluggable buttons
 * Convert the buttons into menu items by taking the button's text, icon and action listener.